.gitignore

```diff
@@ -1,7 +1,24 @@
-node_modules
```

Comment:
I basically used the autogenerated .gitignore and the just added the backend/node_modules